script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_3.txt

```diff
@@ -93,52 +93,52 @@ For some reason, Senpai smiles happily.
 "?"
 Arihiko and I look at each other and tilt our heads.
 <Page29>
-"I envy you. I admire friends who can understand each other like that without any hard feelings."
+"I really envy you. I wish I had friends who can understand eachother without all the fervor."//「羨ましいなあ。そうやって何の気負いもなしで分かり合える友人がいるのって、憧れます」"really" added to make it more casual
```

Comment:
Should this be a "friends who **could** understand **each other**"? Can -> could since it's potential form for the friends, and also not sure that eachother as a single word is valid.